Ex4.py

```diff
@@ -0,0 +1,13 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов.
+# Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок,
+# если известно, что Петя и Сережа сделали одинаковое количество журавликов,
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+n = int(input('Введите количество всех журавликов '))
+if n % 6 == 0:
+    print(n // 6, 2 * (2 * (n // 6)), n // 6)
```

Comment:
Тоже верно. Кроме того, обратите внимание, что у вас трижды повторяется одна и та же операция, то есть ресурсы машины, на которой вы работаете, расходуются не оптимально. Чтобы такого не происходило используют переменные.
```suggestion
    var = n // 6
    print(var, var * 4, var)
```

Reply:
Спасибо, только вникаю в язык :)